utils/xbfmg.c

```diff
@@ -99,12 +99,12 @@ void prtusage( char *prgnam ) {
  *      (last#) = total number of BUFR messages in bufrfile
  * </pre>
  *
- * <b>Program history log:</b>
- * | Date | Programmer | Comments |
- * | -----|------------|----------|
- * | 2018-03-01 | J. Ator |  Original author |
- * | 2021-09-29 | J. Ator |  Use basename instead of pid in output filenames |
- * | 2021-10-08 | J. Ator |  Simplify bvstr instantiation and initialization |
+ * @param argc argument count.
+ * @param argv argument array.
+ *
```

Comment:
I really think these are unnecessary and don't add any value to the documentation.  The Usage block makes it very clear how to call the utility.

Reply:
Yes, but they are needed to suppress warnings.